.gitignore

```diff
@@ -720,3 +720,4 @@ MigrationBackup/
 # Ignore `example` folder, but not its `README.md`
```

Comment:
See previous comment about adding R files to .gitignore.

Reply:
Should be resolved here. https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/28debd8b0eb80a00e98bcf75a8a75f3f052f3274